ChangeLog.txt

```diff
@@ -40,6 +40,13 @@ MINOR:
 - Monatists and Donatists can no longer use the seduction focus, per their religious beliefs
 - Luciferians now praise the Dawn Star and reject ignorance
 - Taoists can no longer hunt
+- Fixed issue with Root West Reformation where Anxi fails to get a working temple, preventing instant Reformation
+- Mimamsa's "incredibly powerful priests" are now actually powerful, but due to the hardcoding of Autocephalacy, which would be realistic for them, a temporary implementation of the Antipope and Investiture mechanics exists to represent the struggle between the Indian kingdoms and their Mimamsa clergy, who vie for control over the monastaries of the realm
```

Comment:
Defining patriarches will be possible in the next update. Antipope and Investiture mechanics are not fitting for this religion.